packages/flutter/lib/src/material/elevated_button.dart

```diff
@@ -83,12 +83,12 @@ class ElevatedButton extends ButtonStyleButton {
   /// The [icon] and [label] arguments must not be null.
   factory ElevatedButton.icon({
     Key key,
```

Comment:
key can be null too according to the ctr above.

Reply:
Thanks, fixed.